### DISCUSSION TITLE
deprecated

### DISCUSSION THREAD

**Discussion by giscus** [Reverier-Xu/discussions#135] [Announcements]
# 二次元是怎么旅游的.jpg - Reverier's Blog

小小记录一下。 因为人已经回来很久了，博客里的照片都是带GPS等exif信息的，方便以后看地方； 这里放了很多很多图，小心流量； 由于博客不方便上

https://blog.woooo.tech/posts/travel_jpn/

**Comment by Reverier-Xu** [comment]
test
